### PR TITLE
Adds encrypted token for github releases and releases 0.1.0

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-### unreleased
+### 0.1.0
     
-**Commit Delta**: [Change from 0.0.0 release](https://github.com/plus3it/terraform-aws-org-new-account-iam-role/compare/0.0.0...unreleased)
+**Commit Delta**: N/A
```
Comment (kbalk) [inline CHANGELOG.md:9]: ?

Comment (lorengordon) [reply CHANGELOG.md:9]: There is no delta link, because 0.0.0 was not released

Comment (kbalk) [reply CHANGELOG.md:9]: I see.

**Review by kbalk — APPROVED** [review]
Looks good.

**Pull request merged**